packages/graphiql/src/components/QueryEditor.tsx

```diff
@@ -83,210 +108,172 @@ export class QueryEditor extends React.Component<QueryEditorProps, {}>
     require('codemirror-graphql/jump');
     require('codemirror-graphql/mode');
 
-    const editor: CM.Editor = (this.editor = CodeMirror(this._node, {
-      value: this.props.value || '',
+    const editorEl = nodeRef.current;
```

Comment:
Do you not want to check whether nodeRef.current is defined here and exit early if it is not yet defined?

Reply:
You'd also need to add nodeRef.current to dependency list in this hook.

Reply:
@ncthbrt done! i think?